massive-data-import/README.md

```diff
@@ -0,0 +1,200 @@
+    count++;
+    console.log(`[${count}] ${notification.document.content.passenger_count} passengers just arrived, and paid ${notification.document.content.fare_amount}$`);
+  })
+  .onDone(() => console.log('Subscribed. Waiting for passengers...'));
+```
+
+After that we are going to import our documents.  
+The documents to be inserted will have to be collected in an array before being passed to the [SDK method](https://docs.kuzzle.io/sdk-reference/collection/mcreate-document/) corresponding to the mCreate action of the Document Controller :
+
+```js
+const documents = [
+  {
+    pickup_datetime:  '03/01/2016 12:10:29 AM',
+    dropoff_datetime: '03/01/2016 12:19:05 AM',
+    passenger_count:  '5',
+    trip_distance:    '2.51',
+    pickup_position:  { lon: '-73.993759155273438', lat: '40.718101501464844' },
+    dropoff_position: { lon: '-73.951248168945312', lat: '40.711910247802734' },
+    fare_amount:      '11.3'
+  },
+  {
+    pickup_datetime:  '03/01/2016 12:10:28 AM',
+    dropoff_datetime: '03/01/2016 12:21:56 AM',
+    passenger_count:  '1',
+    trip_distance:    '2.5',
+    pickup_position:  { lon: '-73.974746704101563', lat: '40.793346405029297' },
+    dropoff_position: { lon: '-73.979293823242187', lat: '40.761508941650391' },
+    fare_amount:      '11.8'
+  }
+]
+
+kuzzle
+  .collection('yellow-taxi', 'nyc-open-data')
+  .mCreateDocumentPromise(documents)
+  .then(() => {
+    console.log(`${documents.length} lines inserted`);
+  })
+  .catch(error => {
+    if (error.status = 206) {
+      console.error(`PartialError: ${error.errors.length} documents insertion fail`);
+    } else {
+      console.error('Error: ');
+      console.dir(error, {colors: true, depth: null});
+    }
+  });
+```
+
+## Try it yourself
+
+You can use the [docker-compose.yml](docker-compose.yml) included in this How-To to test the massive data import.  
+The containers are preconfigured to work with NYC Open Data's Yellow Taxi dataset.
+
+```bash
+docker-compose up
+```
+
+Then in another terminal we are going to subscribe to a room with [geoBoundingBox](https://docs.kuzzle.io/kuzzle-dsl/terms/geo-bounding-box/) corresponding to the Time Square area. We will receive a notification each time a new document corresponds to a passenger dropped in this area.  
+
+```bash
+docker-compose exec kuzzle node /scripts/subscribe.js
+```
+
+In another terminal we are going to import the dataset with the two methods.
+
+```bash
+time docker-compose exec kuzzle node /scripts/loadBulk.js
+
+time docker-compose exec kuzzle node /scripts/loadMCreate.js
+```
+
+On a laptop with a I5-7300U CPU @ 2.60 GHz, 16GiB of RAM and a SSD it takes approximately 1 minute to load 1 million documents in Kuzzle using the Bulk Api method and approximately 2 minutes to load the same number of documents with the mCreate method.  
+This is 40% faster than the cluster result because we don't have any latency.
```

Comment:
The `40% faster` part is confusing because of the mCreate method being `40% slower` than the bulk one (unfortunate coincidence)

Moreover, I don't know if that information is relevant. Once again, I don't think that providing numbers obtained on a laptop environment is helpful in any way.
But if you do insist on adding them, I'd change that sentence slightly, for instance: `(these numbers are higher than the cluster ones because there is no network latency)`

Reply:
I will remove this part, this is more clear without it.